app/src/main/res/values/donttranslate.xml

```diff
@@ -36,6 +36,8 @@
     <string name="settings_autorewind_default" translatable="false">0</string>
     <string name="settings_immediate_playback_key" translatable="false">immediate_playback</string>
     <string name="settings_immediate_playback_default" translatable="false">false</string>
+    <string name="settings_duck_audio_key" translatable="false">false</string>
```

Comment:
The key should be something that can be used to identify the setting. In this case `duck_audio` would be a proper key.

Reply:
Ahh, thanks, must of been a brain fart while I was filling that in.  Thank you for reviewing, it should be fixed now.